dist/gcc-compatible/configure

```diff
@@ -227,7 +227,6 @@ show_help () {
   printf "                    * aarch64-apple-ios\n"
   printf "                    * x86_64-apple-ios-simulator\n"
   printf "    --disable-bzero Do not use explicit_bzero (binary will work with an old GLIBC)\n"
-  printf "    --disable-ocaml Disable OCAML bindings\n"
```

Comment:
I'm not sure whether some folks would prefer keeping the option to build the raw package here.
If yes, the `hacl-star-raw.opam` should be thinned out to make clear that it's not the real one and we should keep the infrastructure here.
But I'd be fine with removing this here as you've done.

Reply:
I don't know, @msprotz what do you think?

Reply:
I don't have a strong opinion either way... my hunch is if I'm working on the bindings I'll almost certainly end up with hacl-star + hacl-packages side-by-side with my testing loop doing build hacl, update hacl in packages, and build packages... so I don't think it makes sense to maintain an old build system in the hacl-star repository 

let's just make sure we're all very good about catching breakages... if the situation changes, we can always reintroduce some amount of build in HACL*